app/javascript/dashboard/store/modules/conversations/actions/messageReadActions.js

```diff
@@ -0,0 +1,35 @@
+import { throwErrorMessage } from 'dashboard/store/utils/api';
```

Comment:
Created a new file as the action.js has passed 300 lines. Modularizing parts of the code would be required.